components/MyDropzone.js

```diff
@@ -40,34 +41,27 @@ class MyDropzone extends React.Component {
         identity: identity
       })
 
-      console.log("got identity:", identity)
-
       this.setState({loadingMessage: "fetching bucket keys"})
```

Comment:
consider changing bucket to storage for user interpretability? not every user will know what a bucket is